Commands/balance.go

```diff
@@ -0,0 +1,12 @@
+package Commands
+
+import (
+
+)
```

Comment:
This isn't needed (and also suggests your editor is not running `go fmt` automatically).

```suggestion
```

Reply:
Fixed